api/src/EventListener/RequestTransactionListener.php

```diff
@@ -56,8 +71,16 @@ public function commitTransaction(KernelEvent $event): void {
         if (HttpKernelInterface::MAIN_REQUEST != $event->getRequestType()) {
             return;
         }
-        $this->validateTransactionNestingLevel();
-        $this->entityManager->getConnection()->commit();
+        if ($this->transactionLevelStack->isEmpty()) {
```

Comment:
Könnte man das mit einem unit test noch testen?

Reply:
Du meinst, `commitTransaction` ist nur erlaubt, wenn `transactionLevelStack` nicht leer ist?

Falls  ja - diesen Test gibt es:
https://github.com/pmattmann/ecamp3/blob/08dd4c10ba76510bb2d77b465b3eb61a983f8b10/api/tests/EventListener/RequestTransactionListenerTest.php#L137

Reply:
Ne, dass nach rollback noch das event Request::commit kommt, und der RequestTransactionListener dann keine exception schmeisst

Reply:
Aha - verstehe, was du meinst.
Naja, es ist das definierte Verhalten von Synfony. Sollte nicht Synfony einen Test dafür haben?
Macht es sinn das Verhalten von Synfony zu testen?

Reply:
Für den nächsten der RequestTransactionListener ändert wäre es cool, wenn er nicht nochmals in der doku graben müsste, sondern der Test fehlschlägt, wenn er das Verhalten von Symfony.
Kann aber auch ich später machen